pkg/daemon/update.go

```diff
@@ -217,6 +220,37 @@ func addExtensionsRepo(osImageContentDir string) error {
 	return nil
 }
 
+func getOCBinaryPath() (string, error) {
+	if _, err := os.Stat(cliBinPath); err == nil {
+		return cliBinPath, nil
+	}
+
+	cliImage := "registry.svc.openshift.org/origin/4.6:cli"
```

Comment:
We need to pull the from the manifest, the same way as we template all the other images.

Reply:
Right, I'll fix that once OKD tests pass